main/src/cgeo/geocaching/ui/CacheDetailsCreator.java

```diff
@@ -182,6 +183,15 @@ public void addSize(final Geocache cache) {
         }
     }
 
+    public void addAlcMode(final Geocache cache) {
+        Log.e("_AL add mode to view: " + cache.isLinearAlc());
```

Comment:
When you are finished with this PR, everything working, you should remove all those debugging `Log.e(...)` statements before finalizing this PR.

Reply:
Ideally I'd put those into comments but I learned in a previous PR that this is considered bad practice. Would it be ok to leave them in as `Log.d()`

Reply:
`Log.d()` is fine as well, but `Log.e()` will break c:geo for users running c:geo in debug mode (settings => system => debug mode), as every `Log.e()` will terminate c:geo in this case.

Reply:
> `Log.d()` is fine as well, but `Log.e()` will break c:geo for users running c:geo in debug mode (settings => system => debug mode), as every `Log.e()` will terminate c:geo in this case.

ahhh ... didn't know that. Thx for telling me.

Reply:
No problem. `Log.e()` is very convenient during development, as you can easily filter for that in LogCat window and "traffic" for `Log.e()` is way less than for `Log.d()`. So my usual way is to use `Log.e()` during development and either remove it or switch it to `Log.d()` or `Log.i()` before finalizing my PR, depending on whether I want to see those messages always (`Log.i()`) or only when the user has set c:geo into debugging mode (`Log.d()`).